Delcam.ProductInterface.PowerShape/Entities/Wireframe/PSGenericCurve.cs

```diff
@@ -216,6 +216,14 @@ public void DeleteCurvePoints(int[] pointsIndeces)
             {
                 throw new Exception("Number of points must be 2 or more");
             }
+            if (startPoint > endPoint)
+            {
+                throw new Exception("Start Point must proceed End Point");
+            }
+            if (startPoint == endPoint & !IsClosed)
```

Comment:
How can they be the same if the curve is closed? Not sure how that works?

Reply:
Yeah, granted it's weird: It's span based, you're specifying the repointing to occur in a range. For example if I have a 5 point closed curve, I have 5 spans: 1-2, 2-3, 3-4, 4-5, 5-1. The whole curve is therefore the span 1-1, or any other index that starts and ends at the same place.